<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="slicesLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Slices</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mirrorBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Mirror</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showGridBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show grid</string>
     </property>
    </widget>
    <widget class="QSlider" name="gridIntensitySLider">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid intensity</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sizeBox">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>640x360</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>800x600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280x1024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1920x1080</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="slicesCount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSlider" name="slicesCountSlider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>751</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="MandalaPainter" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>640</width>
       <height>360</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>470</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Line width</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="lineView">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSlider" name="lineSlider">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="lineColorButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Line color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backgroundButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="colorChanging">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>140</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Color changing</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="action_Open"/>
    <addaction name="action_New"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MandalaPainter</class>
   <extends>QWidget</extends>
   <header location="global">MandalaPainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
